### PR TITLE
Fix documentation of "Root Build script" [JENKINS-38479]

### PR DESCRIPTION
**Pull request by kdanW**
This resolve the issue [JENKINS-38479]

https://issues.jenkins-ci.org/browse/JENKINS-38479

### CODE REVIEW

**Comment by kdanW** [comment]
@jenkinsci/hacktoberfest

**Comment by kdanW** [comment]
No problem. How can i get this merged? I see the checks are failing

File: src/main/resources/hudson/plugins/gradle/Gradle/help-rootBuildScriptDir.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div>
     If your workspace has the top-level build.gradle in somewhere other than the module root directory, specify the path
     (relative to the module root) here,
-    such as ${workspace}/parent/ instead of just ${workspace}. If left empty, defaults to build.gradle
+    as /parent/ instead of just ${WORKSPACE} where ${WORKSPACE}/parent is the absolute. If left empty, defaults to build.gradle in ${WORKSPACE} directory.
```
Comment (wolfs) [inline src/main/resources/hudson/plugins/gradle/Gradle/help-rootBuildScriptDir.html]:
```suggestion
    as parent where ${WORKSPACE}/parent is the absolute. If left empty, defaults to build.gradle in ${WORKSPACE} directory.
```

`/parent` will reference to the absolute path. It should be `parent` instead.

**Comment by kdanW** [comment]
@wolfs can you check now?

**Comment by wolfs** [comment]
Thank you @kdanW!

**Pull request merged**